src/OrchardCore.Commerce.Web/Startup.cs

```diff
@@ -18,11 +18,6 @@ public class Startup
 
     public void Configure(IApplicationBuilder app, IHostEnvironment env)
     {
-        if (env.IsDevelopment())
-        {
-            app.UseDeveloperExceptionPage();
-        }
-
```

Comment:
[This is no longer needed in .NET 6](https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-6.0&tabs=visual-studio#:~:text=in%20the%20preceding%20code%2C%20the%20if%20(env.isdevelopment())%20block%20is%20removed%20because%20in%20development%20mode%2C%20the%20developer%20exception%20page%20middleware%20is%20enabled%20by%20default.%20for%20more%20information%2C%20see%20differences%20between%20the%20asp.net%20core%205%20and%206%20hosting%20models%20in%20the%20next%20section.)

Reply:
Hmm for some reason it didn't worked without it, so I added it back.